config/schedule.yml

```diff
@@ -3,6 +3,11 @@ clear_emergency_login_keys:
   class: 'ClearEmergencyLoginKeysJob'
   queue: low
 
+destroy_inactive_accounts:
+  cron: '10 8 * * *'
```

Comment:
im no an expert with cron syntax but its ok that this has 8 when other below has 09 (instead of 9)?

Reply:
some do some don't. 

https://crontab.guru/

I agree we should standardise on one approach, but it's a very minor thing